modules/map/hdmap/adapter/xml_parser/junctions_xml_parser.cc

```diff
@@ -40,6 +40,7 @@ Status JunctionsXmlParser::Parse(const tinyxml2::XMLElement& xml_node,
       return Status(apollo::common::ErrorCode::HDMAP_DATA_ERROR, err_msg);
     }
 
+
```

Comment:
Don't need this blank line.

Reply:
I have corrected code styles with the clang format tool。